app/messaging/index.js

```diff
@@ -2,10 +2,12 @@ const config = require('../config')
 const processProcessingMessage = require('./process-processing-message')
 const processSubmitMessage = require('./process-submit-message')
 const processReturnMessage = require('./process-return-message')
+const processStatementMessage = require('./process-statement-data-message')
```

Comment:
Maybe rename this `processStatementDataMessage` to be clear what part of the statement service it is and to be consistent with the module name.